mmv1/third_party/terraform/services/compute/resource_compute_instance_test.go.tmpl

```diff
@@ -1857,6 +1857,13 @@ func TestAccComputeInstance_nictype_update(t *testing.T) {
 						t, "google_compute_instance.foobar", &instance),
 				),
 			},
+			{
+				Config: testAccComputeInstance_nictype(instanceName, "IDPF"),
```

Comment:
The test is failing because this config uses an `e2-medium` instance and `IDPF` is only supported on bare metal instances, the smallest of which is `c3-standard-192-metal`. Since this is just an enum value rather than a new field I'm OK with reverting the test addition - spinning up an instance of this size regularly is likely to cause us quota headaches in the future.

Reply:
Fully agree, I reverted the test and I rebased changes on top of the current main.